component/src/main/wso2/kafka/producer.bal

```diff
@@ -79,82 +79,83 @@ public type ProducerConfig record {
+    #
+    # + config - configurations related to the endpoint.
     public function init(ProducerConfig config) {
         self.producerConfig = config;
         self.producerActions.init(config);
     }
 
-    documentation { Registers producer endpoint in the service
-        P{{serviceType}} type descriptor of the service
-    }
+    # Registers producer endpoint in the service.
+    #
+    # + serviceType - type descriptor of the service.
     public function register (typedesc serviceType) {
 
     }
 
-    documentation { Starts the consumer endpoint }
+    # Starts the consumer endpoint.
     public function start () {}
 
-    documentation { Returns the action object of ProducerAction }
-    public function getCallerActions () returns (ProducerAction) {
+    # Returns the action object of ProducerAction.
+    #
+    # + returns - Producer actions.
+    public function getCallerActions () returns ProducerAction {
         return producerActions;
     }
 
-    documentation { Stops the consumer endpoint }
+    # Stops the consumer endpoint.
     public function stop () {
         self.producerActions.close();
     }
 
 };
 
-documentation { Kafka producer action handling object }
+# Kafka producer action handling object.
+#
+# + producerHolder -
+# + connectorID -
 public type ProducerAction object {
 
     public map producerHolder;
     public string connectorID = system:uuid();
 
-    documentation { value:"Simple Send action which produce records to Kafka server
-        P{{value}} record contents
-        P{{topic}} topic the record will be appended to
-        P{{key}} key that will be included in the record
-        P{{partition}} partition to which the record should be sent
-        P{{timestamp}} timestamp of the record, in milliseconds since epoch
-    }
+    # Simple Send action which produce records to Kafka server.
+    #
+    # + value - record contents.
+    # + topic - topic the record will be appended to.
+    # + key - key that will be included in the record.
+    # + partition - partition to which the record should be sent.
+    # + timestamp - timestamp of the record, in milliseconds since epoch.
     public extern function send(byte[] value, string topic, byte[]? key = (), int? partition = (), int? timestamp = ());
 
-    documentation { Flush action which flush batch of records }
+    # Flush action which flush batch of records.
     public extern function flush();
 
-    documentation { Close action which closes Kafka producer }
+    # Close action which closes Kafka producer.
     public extern function close();
 
-    documentation { GetTopicPartitions action which returns given topic partition information
-        P{{topic}} topic which partition information is given
-        R{{}} partition for given topic
-    }
+    # GetTopicPartitions action which returns given topic partition information.
+    # + topic - topic which partition information is given.
+    # + returns - partition for given topic.
```

Comment:
This should be `partition(s) for the given topic`

Reply:
Fixed in 8e52317